mflowgen/icovl/construct.py

```diff
@@ -275,10 +275,11 @@ def construct():
     {'order': [
       'main.tcl','quality-of-life.tcl',
       'stylus-compatibility-procs.tcl','floorplan.tcl','io-fillers.tcl',
+      'alignment-cells.tcl',
+      'gen-bumps.tcl', 'route-bumps.tcl',
+      'sealring.tcl',
```

Comment:
If this flow is never going to run past init, adding the sealring here is fine. Otherwise, it shouldn't be added until the very end.

Reply:
Correct, this node is designed to never run farther than init. I was too lazy to remove the unused steps---could do that now if you want, and/or could make a note to clean it up later.

Reply:
Not a huge deal, but I think it would be good If you could do it now before merging.

Reply:
I got rid of the extra steps. There's a lot of commented-out code that should maybe be deleted at some point. Pending final check, though, which I expect will pass, I'd like to just put that on a to-do list and go ahead with the pull so I can move on to more time-sensitive tapeout issues (like maybe rdl blockages...)